config/locales/pt-BR.yml

```diff
@@ -198,6 +274,51 @@ pt-BR:
   pagination:
     next: Next
     prev: Prev
+    truncate: "&hellip;"
+  remote_follow:
+    acct: Entre seu usuário@domínio do qual quer seguir
+    missing_resource: Não foi possível achar a URL de redirecionamento para sua conta
+    proceed: Prossiga para seguir
+    prompt: 'Você vai seguir:'
   settings:
+    authorized_apps: Aplicativos autorizados
+    back: Voltar ao Mastodon
     edit_profile: Editar perfil
     preferences: Preferências
+    export: Importar dados
+    import: Importar
+    preferences: Preferências
+    settings: Settings
+    two_factor_authentication: Autenticação Two-factor
+  statuses:
+    open_in_web: Abrir no browser
+    over_character_limit: limite de caracter excedeu %{max}
+    show_more: Mostrar mais
+    visibilities:
+      private: Mostrar apenas para seguidores
+      public: Público
+      unlisted: Público, mas não mostre no timeline público
+  stream_entries:
+    click_to_show: Clique pra mostrar
+    reblogged: boosted
+    sensitive_content: Conteúdo sensível
+  time:
+    formats:
+      default: "%d %b, %Y, %H:%M"
+  two_factor_authentication:
+    code_hint: Entre o código gerado pelo seu aplicativo para confirmar
+    description_html: Se você habilitar <strong>autenticação two-factor </strong>, quando logar será necessário o seu telefone que vai gerar os tokens usado.
+    disable: Disabilitar
+    enable: Habilitar
+    enabled_success: Autenticador Two-factor habilitador com sucesso
+    generate_recovery_codes: Gerar códigos para recuperar conta
+    instructions_html: "<strong>Scaneie este código QR no seu Google Authenticator ou aplicativo similar no seu telefone</strong>. A partir de agora seu aplicativo irá gerar tokens que deverão ser digitados para você logar".
```

Comment:
the period here has to be inside of the quotation marks, or else it's a syntax error. the quotation marks aren't part of the string, they're just part of the yml literal syntax